pkg/etcd/etcd.go

```diff
@@ -1067,6 +1114,13 @@ func (e *ETCD) manageLearners(ctx context.Context) {
 			logrus.Warnf("Failed to list nodes with etcd role: %v", err)
 		}
 
+		for _, node := range nodes {
+			_, _, err := e.setEtcdStatusConditionIfNodeIsNotReady(node, client.CoreV1())
```

Comment:
This appears to be the only use of this function, why does it return 2 values that aren't used? If it doesn't have any other callers that need these return values, don't return them.